misc/package/Data Files/MWSE/core/mcm/main.lua

```diff
@@ -222,23 +326,55 @@ local function onClickModConfigButton()
 		modList.widthProportional = 1.0
 		modList.heightProportional = 1.0
 		modList:setPropertyBool("PartScrollPane_hide_if_unneeded", true)
+		-- modList.paddingLeft = 5
 
-		-- Get a sorted list of mods.
-		local sortedConfigModNames = {}
-		for name, package in pairs(configMods) do
-			-- Allow package.hidden to be set to prevent it from showing up in the list.
-			if (not package.hidden) then
-				table.insert(sortedConfigModNames, name)
+		-- make a list of config mods instead of config mod names, so we can sort by name and `favorite` status
+		local configModsList = {} --- @type mwseModConfig[]
+		for _, package in pairs(configMods) do
+			if not package.hidden then
+				table.insert(configModsList, package) 
 			end
 		end
-		table.sort(sortedConfigModNames, caseInsensitiveSorter)
+
+		table.sort(configModsList, sortPackages)
 
 		-- Fill in the mod list.
 		local modListContents = modList:getContentElement()
-		for i = 1, #sortedConfigModNames do
-			local modName = sortedConfigModNames[i]
-			local entry = modListContents:createTextSelect({ id = "ModEntry", text = modName })
+
+		for _, package in ipairs(configModsList) do
+			local entryBlock = modListContents:createBlock{id="ModEntryBlock"}
```

Comment:
Can you add some spaces before and after `=` and between the arguments and braces?